### ISSUE TITLE
Require astroid >=2.3.0 for pylint 2.4.0

### ISSUE THREAD

**Issue by PhilippSelenium** [PyCQA/pylint#3117]
Pylint 2.4.0 is not compatible to astroid versions < 2.3.0 which should be reflected in
https://github.com/PyCQA/pylint/blob/master/pylint/__pkginfo__.py#L32

### Steps to reproduce
1. Install 
```
pip install astroid==2.2.5
pip install pylint==2.3.1
```
3. Update
```
pip install pylint -U
```
2. Check
```
pylint test.py
```

contents of test.py
```
def foo():
    return "bar"
```

### Current behavior
astroid stays at version > 2.2.5 which leads to:
'Arguments' object has no attribute 'type_comment_args'

### Expected behavior
astroid is updated to version compatible to pylint 2.4.0

### pylint --version output
depends ;)

**Comment by naslundx** [comment]
Maybe related: Looks like pylint 2.3.x allows astroid 2.3, but it is giving me a lot of "weird" false positives I wasn't getting with 2.2, such as:

```
color.py:19:55: E1101: Instance of 'Color' has no 'r' member; maybe 'g'? (no-member)
color.py:19:63: E1101: Instance of 'Color' has no 'g' member; maybe 'r'? (no-member)
```

**Comment by PCManticore** [comment]
Just fixed with #3130 will be part of 2.4.1 shortly.
